generate-bookmarks-html

```diff
@@ -156,7 +156,7 @@ cat <<EOD
 html {
     max-width: 60em;
     margin: auto;
-    font-family: "Calibri", sans-serif;
+    font-family: Calibri, sans-serif;
 }
```

Comment:
You’re completely right, but this is a matter of preference for me. (I’ll be saying that a lot.) I find it easier to only leave the predefined font families (e.g. `sans-serif`, `monospace`) unquoted.